.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,16 @@
+name: GitHub Actions Demo
+on: [push]
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ls -l
+      - run: docker build --target test --tag todo-app:test .
+      - name: Docker Run
+        env:
+          TRELLO_API_KEY: ${{secrets.TRELLO_API_KEY }}
+          TRELLO_API_SECRET: ${{secrets.TRELLO_API_SECRET }}
+          TRELLO_BOARD_ID: ${{secrets.TRELLO_BOARD_ID }}
+        run: docker run --env TRELLO_API_KEY=$TRELLO_API_KEY --env TRELLO_API_SECRET=$TRELLO_API_SECRET --env TRELLO_BOARD_ID=$TRELLO_BOARD_ID -p 5000:5000 todo-app:test
```

Comment:
You don't need real Trello details for your unit test because it shouldn't actually be talking to Trello. So we shouldn't add in the real secrets because that just increases the chance of it accidentally modifying real data or leaking credentials.

You would only need to add real credentials like this for e2e tests.

You also don't need to publish a port (`-p 5000:5000`) because you're not really running or viewing the todo app.

So this can just be `run: docker run todo-app:test`